Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -831,59 +831,38 @@ def com_google_fonts_check_metadata_category(family_metadata):
         yield PASS, "OK!"
 
 
-@disable  # TODO: re-enable after addressing issue #1998
 @check(
-    id="com.google.fonts/check/family/equal_numbers_of_glyphs",
+    id="com.google.fonts/check/family/equal_codepoint_coverage",
     conditions=["are_ttf", "stylenames_are_canonical"],
     proposal="legacy:check/011",
 )
```

Comment:
Thanks, @simoncozens

Since this is essentially a new check, I think it would be good to set `experimental = "Since 2023-Oct-13"` and give it a week or so for onboarders to provide feedback